src/jit/simd.h

```diff
@@ -49,4 +49,14 @@ struct SIMDIntrinsicInfo
 
 #endif // FEATURE_SIMD
 
+enum SIMD_Level
```

Comment:
I'd like to see a good comment here explaining: (1) what this is used for, (2) what "Min" supported means, (3) what the rules are, e.g.: are these in strictly increasing order?

Also, how do they relate to getSIMDInstructionSet()?

Reply:
E.g., it seems like your comment from https://github.com/dotnet/coreclr/issues/14065 should be in the code someplace, in some appropriate form.

Reply:
Got it.